html/profile.html

```diff
@@ -10,9 +10,91 @@
 </head>
 
 <body>
-  <h1 class="title-of-page">Setting up Profile.</h1>
 
-  <script src="../script/profile.js"></script>
+  <div class="app">
+
+    <div class="app__body">
+
+      <div class="sidebar">
+
+        <div class="sidebar__header">
+          <img src="../image/img_avatar.png" alt="avtar" class="avatar">
+          <button class="button" style="vertical-align:middle"><span>Name</span>
+        </div>
+
+        <div class="sidebar__search">
+          <div class="sidebar__searchContainer">
+            <img src="../image/search.png" alt="search" class="search">
+            <input placeholder="Search" type="search">
+          </div>
+        </div>
+
+        <div class="sidebar__chat">
```

Comment:
`sidebar__friend-list` can be a good name i think ?

Reply:
changed it to sidebar__friend-list